### DISCUSSION TITLE
Copilot Plugin Broken for Jetbrain Gateway products

### DISCUSSION THREAD

**Discussion by paulJonesCalian** [community/community#109714] [Copilot Conversations]
### Select Topic Area

Bug

### Body

On most Jetbrains products, you install Copilot once to get all features working. On the Jetbrains Gateway product, you need to install it twice.

1) If you install on the Client, you get code completion working as expected
2) If you install on the Host, you get CoPilot Chat
3) If you install on both, you get chat, the completions show up, but hitting tab does not fill in the completion.

Could you fix the 3rd scenario? Or at least update the plugin, so that the plugin is only required on the host or client?

**Comment by kalyan02** [comment]
This crash is also causing IDE/Editor to stop working. I have to restart client and reconnect to remote server every single time.

**Reply by pauljones0** [reply]
I just installed 2024.1 Beta this morning and BOTH chat and tab completion appears to be working, when CoPilot is installed in the client plugins. Not holding my breath, but neither has broken for me yet...

**Comment by MC-XiaoHei** [comment]
In my tests, you need to install the plugin only in the client and open the window with Open Copilot Chat in the menu, which worked for me

**Reply by prestopUCAR** [reply]
Which menu are you using that has "Open Copilot Chat"?

**Reply by Nm774499** [reply]
> Which menu are you using that has "Open Copilot Chat"?

**Reply by MC-XiaoHei** [reply]
right click any file

**Comment by robertpro** [comment]
Is there any update for this issue? any work around? this is called "community" but I am paying for this product so I would like to get any estimation to get this solved.
